boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -105,7 +106,7 @@ hardware features:
 | WDT       | on-chip    | watchdog             |
 +-----------+------------+----------------------+
 
-The nrf5340pdk_nrf5340_cpunet board configuration supports the following
+The nrf5340dk_nrf5340_cpunet board configuration supports the following
 hardware features:
 
 +-----------+------------+----------------------+
```

Comment:
Line 146/147 should be nRF5340 DK (not Development Kit)

Reply:
Yeap, updated all ARM to Arm.